components/Handlers.js

```diff
@@ -0,0 +1,44 @@
+import { render } from '@asyncapi/generator-react-sdk';
+import { pascalCase } from './common';
+
+
+let subscriptionFunction = (channelName, operation, message) => `
+// ${operation} subscription handler for ${channelName}.        
+func ${operation}(msg *message.Message) error {
+    log.Printf("received message payload: %s", string(msg.Payload))
+
+    var lm ${message}
+    err := json.Unmarshal(msg.Payload, &lm)
+    if err != nil {
+        fmt.Printf("error unmarshalling message: %s, err is: %s", msg.Payload, err)
```

Comment:
Shouldn't this be using `log` instead of `fmt` ?

Reply:
you are right fixing this